docs/configuration.rst

```diff
@@ -271,3 +271,21 @@ Zend Configuration
         ],
     ];
 
+How to Override RunSqlCommand Creation
+-------------------------
+
+The following Laminas configuration can be used to override the `creation
+<../src/DoctrineORMModule/Service/RunSqlCommandFactory.php>`_ of the
+``Doctrine\DBAL\Tools\Console\Command\RunSqlCommand`` instance used by this
+module.
```

Comment:
I'm having a hard time getting the relative link working, but while I investigate that, do you think this will work?

Reply:
I don't think trying to link to the source here is the best solution.  Not linking would be fine.

Reply:
Sure, will do.